tests/pages/firefox/partners.py

```diff
@@ -0,0 +1,168 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage, FirefoxBasePageRegion
+from pages.regions.modal import Modal
+
+
+class PartnersPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/partners'
+    _firefox_os_menu_locator = (By.CSS_SELECTOR, '#os nav')
+    _marketplace_menu_locator = (By.CSS_SELECTOR, '#marketplace nav')
+    _android_menu_locator = (By.CSS_SELECTOR, '#android nav')
+    _phone_overview_screen_locator = (By.ID, 'screen-overview')
+    _phone_os_screen_locator = (By.ID, 'screen-os')
+    _phone_marketplace_screen_locator = (By.ID, 'screen-marketplace')
+    _android_phone_locator = (By.CSS_SELECTOR, '.android-phone-visible')
```

Comment:
More of a question than a suggestion, but which is better to use here: `CSS_SELECTOR` or `CLASS_NAME`?